docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -27,7 +27,8 @@ image::images/fleet-server-hosted-container.png[{fleet-server} hosted agent]
 
 Next modify the {fleet-server} configuration by editing the agent policy: 
 
-. In *Fleet*, click *Agent Policies*. Click on the *{ecloud} agent policy* to
+// lint ignore elastic-cloud
+. In *{fleet}*, click *Agent Policies*. Click on the *Elastic Cloud agent policy* to
```

Comment:
Why is this ignored?

Reply:
This is ignored because of the guideline to override if "The name of a product/feature is used in the name of another product/feature" (in the original post). **BUT** if that doesn't apply here, we can use `{ecloud}` here. I wasn't 100% about the phrase "Elastic Cloud agent policy" because in some places it was "Elastic Cloud Agent Policy", which made me think it was it's own product/feature.

I'm sure there are other phrases like this that could use a second set of (more experienced) eyes!

Reply:
It should be Elastic Cloud agent policy to match the policy name as it appears in the UI:

![image](https://user-images.githubusercontent.com/14206422/165398663-6fc087db-cd7c-4513-ae9f-5d62fa2a9aa6.png)

Reply:
It sounds like `{ecloud} agent policy` is ok to use. If we want to create a new attribute or not use `{ecloud}` we can revert https://github.com/elastic/observability-docs/pull/1820/commits/98ca5a53f864b5daf5818085fa080efd0f3ba789 and make any necessary edits.